tests/unit/lms/product/factory_test.py

```diff
@@ -125,4 +125,5 @@ def with_mocked_from_request(self):
     def pyramid_request(self, pyramid_request):
         pyramid_request.matched_route = Mock(spec_set=["name"])
         pyramid_request.matched_route.name = "some.route"
+        pyramid_request.lti_params = {}
```

Comment:
Fixed the fixture to have values for this.

Here we were not expecting them so needs fixing.